DOC: add english doc 'Pod scheduling optimization' (#2409)
fluid-cloudnative/fluid: docs/en/operation/pod_schedule_global.md

> # Demo - Pod Scheduling Optimization

Author: Should I change the title to just 'Optimization', or keep the title same as the file name (Pod Schedule Optimization)? Please be more detailed, thanks.

          weight: 100
```

As affected by affinity, the Pod is scheduled to the node.172.16.0.16 where there is no cache (i.e. no Alluxio Worker Pod running):

```shell
$ kubectl get pods nginx-1 -o  custom-columns=NAME:metadata.name,NODE:.spec.nodeName
NAME    NODE
nginx-1   node.172.16.0.16
```

## Running Demo 2: Create a Pod with mounted dataset, which will try to schedule to the node where the mounted dataset exists

**Create a Pod**

```shell
$ cat<<EOF >nginx-2.yaml
apiVersion: v1
kind: Pod
metadata:
  name: nginx-2
spec:
  containers:
    - name: nginx-2
      image: nginx
      volumeMounts:
        - mountPath: /data
          name: hbase-vol
  volumes:
    - name: hbase-vol
      persistentVolumeClaim:
        claimName: hbase
EOF
$ kubectl create -f nginx-2.yaml
```

**Check the Pod**

Checking the yaml file of Pod, shows that the following information has been injected:

```yaml
spec:
  affinity:
    nodeAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - preference:
          matchExpressions:
          - key: fluid.io/s-default-hbase
            operator: In
            values:
            - "true"
        weight: 100
```

> Through the Webhook mechanism, the application Pod is injected and caches the weak affinity configuration of the worker.

Reviewer: ```suggestion
Through the Webhook mechanism, the application Pod is injected with preferred affinity to the cache worker.
```
"preferred" is recommended here.
